.github/workflows/ci-tests.yml

```diff
@@ -303,7 +303,9 @@ jobs:
               working-directory: /home/nominatim/nominatim-project
 
             - name: Import
-              run: nominatim import --osm-file ../test.pbf
+              run: |
+                  nominatim import --prepare-database --osm-file ../test.pbf
```

Comment:
`--prepare-database` shouldn't need the OSM files.

Reply:
Added an extra check in the cli and changed the ci-tests.